07-studio/01-quickstart.md

```diff
@@ -0,0 +1,295 @@
+match $file isa file, has file-name "concept/type/attributetype.feature"; $commit isa commit, has commit-hash "6e462bcbef73c75405264777069a22bca696a644";insert $commit-file(commit: $commit, file: $file) isa commit-file;
+match $file isa file, has file-name "concept/type/entitytype.feature"; $commit isa commit, has commit-hash "6e462bcbef73c75405264777069a22bca696a644";insert $commit-file(commit: $commit, file: $file) isa commit-file;
+match $file isa file, has file-name "concept/type/relationtype.feature"; $commit isa commit, has commit-hash "6e462bcbef73c75405264777069a22bca696a644";insert $commit-file(commit: $commit, file: $file) isa commit-file;
+match $author isa user, has user-name "haikalpribadi"; $repo isa repo, has repo-name "typedb-behaviour"; insert $commit isa commit, has commit-hash "184bc8a64aa69e383bf496c70b11f02201d33616", has commit-date "Fri May 13 20:24:46 BST 2022"; $commit-author(commit: $commit, author: $author) isa commit-author; $commit-repo(commit: $commit, repo: $repo) isa commit-repo; 
+match $file isa file, has file-name "typeql/language/define.feature"; $commit isa commit, has commit-hash "184bc8a64aa69e383bf496c70b11f02201d33616";insert $commit-file(commit: $commit, file: $file) isa commit-file;
+match $file isa file, has file-name "typeql/language/undefine.feature"; $commit isa commit, has commit-hash "184bc8a64aa69e383bf496c70b11f02201d33616";insert $commit-file(commit: $commit, file: $file) isa commit-file;
+```
+<div class="slideshow">
+
+[slide:start]
+[body:start]![Create Another New File](/docs/images/studio/project-new-file-2.png)[body:end]
+[footer:start]Create another file by clicking the '+' icon again.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![Paste In The Query](/docs/images/studio/project-data-pasted.png)[body:end]
+[footer:start]Copy the above queries into this new file.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![Run the Query](/docs/images/studio/project-data-query-run.png)[body:end]
+[footer:start]Ensure your session and transaction settings are set to `data` and `write` respectively, then click the green play button.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![Commit the Transaction](/docs/images/studio/project-data-committed.png)[body:end]
+[footer:start]Finally, commit your transaction by clicking the green tick.[footer:end]
+[slide:end]
+
+</div>
+
+## Query the Data
+
+Everything is in place and we can start writing queries for our data. Open one final file and set your session and transaction settings to `data` and `read` respectively.
+
+Also ensure that you have turned `infer` on to ensure that rules get resolved. This typically increases the time that queries take to resolve, so it is not enabled by default. However, our schema and queries use a rule.
```

Comment:
I'm thinking we can reword this a little, something like the following?
```suggestion
To the right of the transaction type, we have **transaction settings**, such as `snapshot` and `infer`. Because our query makes use of the `file-collaborator-rule` defined in the schema, you'll need to enable the `infer` option, which allows rules to be triggered when querying.
```